test-clients/src/main/java/com/hedera/services/bdd/suites/contract/traceability/NewTraceabilitySuite.java

```diff
@@ -89,8 +680,35 @@ private HapiApiSpec vanillaBytecodeSidecar() {
                                 .hasKnownStatus(SUCCESS)
                                 .via(firstTxn))
                 .then(
-                        contractBytecodeSidecar(
-                                EMPTY_CONSTRUCTOR_CONTRACT, vanillaBytecodeSidecar, firstTxn));
+                        withOpContext(
+                                (spec, opLog) -> {
+                                    final HapiGetTxnRecord txnRecord = getTxnRecord(firstTxn);
+                                    allRunFor(
+                                            spec,
+                                            txnRecord,
+                                            expectContractActionSidecarFor(
+                                                    firstTxn,
+                                                    List.of(
+                                                            ContractAction.newBuilder()
+                                                                    .setCallType(CREATE)
+                                                                    .setCallOperationType(
+                                                                            CallOperationType
+                                                                                    .OP_CREATE)
+                                                                    .setCallingAccount(
+                                                                            spec.registry()
+                                                                                    .getAccountID(
+                                                                                            GENESIS))
+                                                                    .setRecipientContract(
+                                                                            spec.registry()
+                                                                                    .getContractId(
+                                                                                            EMPTY_CONSTRUCTOR_CONTRACT))
+                                                                    .setGas(197000)
+                                                                    .setGasUsed(66)
+                                                                    .setOutput(EMPTY)
+                                                                    .build())));
+                                }),
+                        expectContractBytecodeSidecarFor(
+                                firstTxn, EMPTY_CONSTRUCTOR_CONTRACT, EMPTY_CONSTRUCTOR_CONTRACT));
     }
 
     HapiApiSpec vanillaBytecodeSidecar2() {
```

Comment:
Could be made private for consistency with the other fn.

Reply:
Changed to `private`